src/libraries/component-scene2d/Scene2d.tsx

```diff
@@ -0,0 +1,312 @@
+import { BaseCanvas, pointInRect, RectangularRegion, Vec4 } from "libraries/figurl-canvas";
+import { dragSelectReducer } from "libraries/util-drag-select";
+import React, { FunctionComponent, useCallback, useEffect, useReducer, useState } from "react";
+
+// https://figurl.org/f?v=gs://figurl/spikesortingview-9&d=sha1://309f3b0131934091301c3e75669138a2e04e9240&label=Test1
+// https://figurl.org/f?v=http://localhost:3000&d=sha1://309f3b0131934091301c3e75669138a2e04e9240&label=Test1
+
+type T = {
```

Comment:
`T` is very commonly used for type parameters, so might be a good style idea to use a different name here. Maybe `Scene2dObjectSubtype[s]`?

Reply:
yes, I changed this in a later version (the scene2d-documentation PR)